tests/test_opds.py

```diff
@@ -549,6 +541,39 @@ def test_adobe_id_tags_when_vendor_id_configured(self):
         self._db.delete(setting)
         assert [] == self.annotator.adobe_id_tags("new identifier")
 
+    def test_lcp_acquisition_link_contains_hashed_passphrase(self):
+        [pool] = self.work.license_pools
+        identifier = pool.identifier
+        patron = self._patron()
+
+        # Setup LCP credentials
+        lcp_credential_factory = LCPCredentialFactory()
+        lcp_credential_factory.set_hashed_passphrase(
+            self._db, patron, "hashed password"
```

Comment:
It might make sense to put it in a separate variable in order to avoid duplication.